src/linalg/mod.rs

```diff
@@ -163,6 +165,32 @@ pub trait BaseVector<T: RealNumber>: Clone + Debug {
     ///assert_eq!(a.unique(), vec![-7., -6., -2., 1., 2., 3., 4.]);
     /// ```
     fn unique(&self) -> Vec<T>;
+
+    /// Computes the arithmetic mean.
+    fn mean(&self) -> T {
+        let n = self.len();
+        let mut mean = T::zero();
+
+        for i in 0..n {
+            mean += self.get(i);
+        }
+        mean / T::from_usize(n).unwrap()
+    }
+    /// Computes the standard deviation.
+    fn std(&self) -> T {
```

Comment:
Does it make sense to have something to calculate the variance? and use here `self.variance().sqrt()`?